README.md

```diff
@@ -87,11 +87,11 @@ from for pull requests.
 ~~~ yaml
 approvers:
     - name1
-    - name2
+    - name2 email-address or any comment
```

Comment:
this should be `name2 # free form text.`

Reply:
yes. ok.